wcfsetup/install/files/lib/action/ModerationQueueAssignUserAction.class.php

```diff
@@ -75,7 +75,7 @@ public function handle(ServerRequestInterface $request): ResponseInterface
 
         $form = $this->getForm($moderationQueue);
 
-        if ($request->getMethod() === 'GET') {
+        if ($request->getMethod() === 'GET' || $request->getMethod() === 'HEAD') {
```

Comment:
This is “not great”, but I'd rather not disallow `HEAD` by default. Not sure what the best course of action is here.